### PR TITLE
Add rotation doc.

### PR DESCRIPTION
**Pull request by jiggoha**

### CODE REVIEW

File: docs/log_rotation.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "bucket": "armored-witness-firmware-log-$ENV-$LOG_SHARD",
+        "kmsKeyName": "ft-log-$ENV",
+        "kmsKeyRing": "firmware-release-$ENV",
+        "kmsKeyVersion": $LOG_SHARD,
+        "kmsKeyLocation": "global",
+        "noteKeyName": "transparency.dev-aw-ftlog-$ENV-$LOG_SHARD"
+    }'
+    ```
+1. Apply [`api_transparency_dev`](/deployment/api_transparency_dev) Terraform to create new https://api.transparency.dev paths for the firmware artefacts and log.
+1. Populate the buckets by running the build triggers.
```
Comment (AlCutter) [inline docs/log_rotation.md:29]: We should probably have a step in here to ensure that all 4 types of firmware are present in the new log before continuing on to update the tooling deps below.

Could be a manual "go click Run on the 4 triggers before continuing", but I imagine we can probably automate that too somehow?

Comment (AlCutter) [reply docs/log_rotation.md:31]: Aha, great!

One suggestion, maybe change the list item to make it clearer about what we're doing here and why, e.g. "Populate the new log with all four firmware types so new devices can straightaway be provisioned from it." or something?

File: docs/log_rotation.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Populate the buckets by running the build triggers.
+    * For CI:
+      ```
+      gcloud builds triggers run applet-build-ci --branch=main
+      gcloud builds triggers run os-build-ci --branch=main
+      gcloud builds triggers run boot-build-ci --branch=main
+      gcloud builds triggers run recovery-build-ci --branch=main
+      ```
+    * For prod, create a new release on the Github repo.
```
Comment (AlCutter) [inline docs/log_rotation.md:39]: This'll have to be all 3 repos I guess? (As an aside, no need to implement this now, but I wonder if it's possible to resolve the latest tags for each and fire `gcloud build`s with that?)

File: docs/log_rotation.md (hunk 1 above)
Comment (jiggoha) [reply docs/log_rotation.md]: Done.

File: docs/log_rotation.md (hunk 2 above)
Comment (jiggoha) [reply docs/log_rotation.md]: Good point, also listed that option, since you don't necessarily need to create a new release.

**Review by AlCutter — APPROVED** [review]
Ace :)

**Pull request merged**